hydradx/tests/test_omnipool_agents.py

```diff
@@ -168,3 +168,38 @@ def test_omnipool_LP(omnipool: oamm.OmnipoolState):
     final_state = events[-1]
     if final_state.agents['agent'].holdings['HDX'] != 0 or events[-2].agents['agent'].holdings['HDX'] == 0:
         raise AssertionError('HDX not invested at the right time.')
+
+
+def test_agent_copy():
+    import random
```

Comment:
This should be at top of file shouldn't it?

Reply:
Moved it to a different file.